docs/command/atlas-federatedAuthentication-federationSettings-describe.txt

```diff
@@ -0,0 +1,86 @@
+.. _atlas-federatedAuthentication-federationSettings-describe:
+
+=========================================================
+atlas federatedAuthentication federationSettings describe
+=========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Return the Federation Settings details for the specified organization.
+
+To use this command, you must authenticate with a user account or an API key with the Organization Owner role.
+
+Syntax
+------
+
+.. code-block::
+   :caption: Command Syntax
+
+   atlas federatedAuthentication federationSettings describe [options]
+
+.. Code end marker, please don't delete this comment
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for describe
+   * - --orgId
+     - string
+     - false
+     - Organization ID to use. This option overrides the settings in the configuration file or environment variable.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see the full output, use the -o json option.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.
+
+Output
+------
+
+If the command succeeds, the CLI returns output similar to the following sample. Values in brackets represent your values.
+
+.. code-block::
+
+   ID     IDENTITY PROVIDER ID   IDENTITY PROVIDER STATUS
+   <Id>   <IdentityProviderId>   <IdentityProviderStatus>
+   
+
+Examples
+--------
+
+.. code-block::
+
+   # Return the JSON-formatted Federation Settings details for the organization with the ID 5e2211c17a3e5a48f5497de3:
```

Comment:
For these examples, I think you can safely remove "for the organization with the ID..."

Reply:
Thanks, I removed!